### ISSUE TITLE
Is there a way to apply border radius to jsPanel or make the whole window transparent?

### ISSUE THREAD

**Issue by inglesuniversal** [Flyer53/jsPanel4#38]
Wonderful project! You can be sure that I will start donating as soon as we roll out the project

Kudos

Hugo Barbosa

**Comment by Flyer53** [comment]
Hi Hugo,
thanks for the praise 😄 

**border-radius**
There is no built-in way currently to apply a border-radius but below you find the code for a extension that allows to add a border-radius depending on the presence of header and/or footer sections.

`jsPanel.extend()` adds a method to the panel. In this case the method `borderRadius(rad)` where **rad** is a number used as pixel value. The method returns the panel so you can chain other methods if needed.
```javascript
jsPanel.extend({
    borderRadius: function (rad) {
        var br = rad+'px';
        var hdr = this.header.style;
        var cont = this.content.style;
        var ftr = this.footer.style;
        // set border-radius of outer div
        this.style.borderRadius = br;
        // set border-radius of either header or content section depending on presence of header
        if (this.querySelector('.jsPanel-hdr')) {
            hdr.borderTopLeftRadius = br;
            hdr.borderTopRightRadius = br;
        } else {
            cont.borderTopLeftRadius = br;
            cont.borderTopRightRadius = br;
        }
        // set border-radius of either footer or content section depending on presence of header
        if (this.querySelector('.jsPanel-ftr.active')) {
            ftr.borderBottomLeftRadius = br;
            ftr.borderBottomRightRadius = br;
        } else {
            cont.borderBottomLeftRadius = br;
            cont.borderBottomRightRadius = br;
        }
        // return the panel
        return this;
    }
});
```
Now you can call the new panel method like
```javascript
jsPanel.create({
    // panel config
}).borderRadius(6);
```
or
```javascript
var p = jsPanel.create();
p.borderRadius(6);
```

**transparent panel**
Take a look at the callback of [example 7 of option content](https://jspanel.de/api.html#options/content/example-7) which creates a transparent panel (except the content). That might be what you're after.

Regards,
Stefan

**Comment by Flyer53** [comment]
@inglesuniversal 
Just for info: the next release might have an `option.borderRadius` to optionally apply a border radius to a panel.

**Comment by Flyer53** [comment]
@inglesuniversal 
Hi Hugo, I just released v4.1.0 which includes a new option `borderRadius`.
